DependencyInjection/Configuration.php

```diff
@@ -68,6 +68,17 @@ public function getConfigTreeBuilder()
                             ->end()
                         ->end()
                     ->end()
+                    ->children()
+                        ->arrayNode('symmetric')
+                            ->useAttributeAsKey('name')
+                            ->prototype('array')
+                                ->children()
+                                    ->scalarNode('path')->isRequired()->end()
+                                    ->booleanNode('base64')->isRequired()->defaultTrue()->end()
```

Comment:
May be it would be reasonable to name this setting like 'binary_output' in order to be similar to corresponding setting in RSA?